app/build.gradle

```diff
@@ -7,8 +7,8 @@ android {
     defaultConfig {
         minSdkVersion 17
         targetSdkVersion 23
-        versionCode 2
-        versionName "2.0"
+        versionCode 3
```

Comment:
Knowing nothing about Gradle, is this normal that the version code is one thing and the version name is something entirely different?